tests/testthat/test-vdiagram.R

```diff
@@ -19,4 +19,20 @@ test_that("vdiagram examples work", {
  expect_is(vdiagram(data, 'html'), 'character')
 })
 
+context("testing vdiagram when gage heights are blank")
+test_that("vdiagram examples work", {
```

Comment:
ok, I guess that makes sense, but how should we catch the case where like this task, we're not handling missing data with a specific report and it fails to render?